iOS/DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -287,18 +294,33 @@ struct NetworkProtectionStatusView: View {
     @ViewBuilder
     private func about() -> some View {
         Section {
-            NavigationLink(UserText.netPVPNSettingsFAQ, destination: LazyView(NetworkProtectionFAQView()))
+            NavigationLink(destination: LazyView(NetworkProtectionFAQView())) {
+                HStack {
+                    Image("Help-24")
+                    Text(UserText.netPVPNSettingsFAQ)
+                }
                 .daxBodyRegular()
                 .foregroundColor(.init(designSystemColor: .textPrimary))
+            }
 
             if statusModel.usesUnifiedFeedbackForm {
-                NavigationLink(
-                    UserText.subscriptionFeedback,
-                    destination: LazyView(UnifiedFeedbackRootView(viewModel: feedbackFormModel))
-                )
+                NavigationLink(destination: LazyView(UnifiedFeedbackRootView(viewModel: feedbackFormModel))) {
+                    HStack {
+                        Image("Support-24")
```

Comment:
These are not rendering correctly in dark mode.  I think you need to change these icons to render as "template" in the assets catalog.